assets/js/signup.js

```diff
@@ -79,6 +68,40 @@ var loadQuestions = function() {
       validationObject["rules"] = rules;
       validationObject["messages"] = messages;
       validateForm(validationObject);
+
+      //Since non-required fields are "valid" when they are empty, we need an
+      //alternate way to keep labels raised when there is content in their
+      //associated input field
+      $( ".vita-form-textfield input" ).focusout(function() {
+        if($.trim($(this).val()).length > 0) {
```

Comment:
I think you're referring to my comment immediately above this one which is a repeat of my comment in the earlier thread. Not sure why it's repeated.